<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="serverIPLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="serverIPLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器端IP地址：</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="usernameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="unLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pwLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pwLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_register">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>311</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="UserNameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="ConfirmLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="UserNameLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="ConfirmLineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="registerBtn">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnMinWindow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCloseWindow">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>×</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnToLogin">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnToRegister">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameMain">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>621</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableView" name="tvSelectDish">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshValiddish">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
    <widget class="QTableView" name="tvSelected">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>191</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRefreshSelectedDish">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新列表</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
